spec/features/visitors/sign_up_spec.rb

```diff
@@ -154,29 +86,21 @@
       expect(current_path).to eq users_otp_path
     end
 
-    it 'disables mobile 2FA after Sign Out if user has no mobile' do
+    it 'disables 2FA after Sign Out if user has no mobile' do
       user = User.find_by_email('test@example.com')
       click_link(t('upaya.headings.log_out'), match: :first)
-      expect(user.reload.second_factors.pluck(:name)).to_not include('Mobile')
+      expect(!user.reload.two_factor_enabled?)
     end
   end
 
-  context 'visitor can confirm mobile 2FA device when email is also selected', email: true do
+  context 'visitor can confirm mobile 2FA device', email: true do
     before do
       sign_up_with_and_set_password_for('test@example.com')
-      check 'Mobile'
       fill_in 'Mobile', with: '555-555-5555'
-      check 'Email'
       click_button 'Submit'
     end
 
-    it 'does not send the OTP via email so the mobile OTP option can be confirmed' do
-      expect(last_email.subject).to eq t('devise.mailer.confirmation_instructions.subject')
-      expect(last_email.body).
-        to match('To finish creating your Upaya Account, you must confirm your email address')
-    end
-
-    it 'informs the user that the OTP code is only sent to the mobile' do
+    it 'informs the user that the OTP code is sent to the mobile' do
```

Comment:
Several of these tests are duplicated in the previous context block. Could you please consolidate them into one context?